src/histolab/tile.py

```diff
@@ -170,7 +189,7 @@ def _has_tissue_more_than_percent(self, tissue_percent: float = 80.0) -> bool:
             True if tissue represent more than ``tissue_percent`` % of the image, False
             otherwise.
         """
-        filters = self._enough_tissue_mask_filters
+        filters = FiltersComposition(Tile).tissue_mask_filters
```

Comment:
thif funcion `_has_tissue_more_than_percent` should be sorted in alphabetical order